examples/index.json

```diff
@@ -1467,6 +1467,13 @@
         "verifyBuild": true,
         "verifyOutput": true,
         "isLearnByExample": true
+      },
+      {
+        "name": "Fork",
+        "url": "fork",
+        "verifyBuild": true,
+        "verifyOutput": false,
```

Comment:
Why is this disabled?

Reply:
The output is may vary for every execution, since we are calling 
```
execute(true),
execute(false)
```

Reply:
Updated the sample https://github.com/ballerina-platform/ballerina-distribution/pull/5045/commits/2424cde4faf9430cb7e1d4f59fe7ca7aea023876